apps/app/navigation/Navigation.tsx

```diff
@@ -199,7 +205,15 @@ function RootNavigator() {
   const dimensions = useWindowDimensions();
 
   return (
-    <Stack.Navigator>
+    <Stack.Navigator
+      screenOptions={{
+        headerStyle: styles.header,
+        headerTitle: (props) => <Heading lvl={3}>{props.children}</Heading>,
```

Comment:
I am not sure if this will stay a _lvl 3_ Heading, as in forms e.g. we will have the context (2) - a form header (3) and maybe a list (4) .. so we need to think about that. Maybe we have our first use-case of looks vs. semantics. So a differentiation between `lvl` and `styling-lvl` might be necessary at some point